docs/README.md

```diff
@@ -0,0 +1,137 @@
+# letsencrypt
+
+This role issues Let's Encrypt certificates via DNS-01 or HTTP-01 challenge.
+
+**Please note that you have to use a dns-challenge if you want to issue wildcard certificates.**
+
+For distribution of certificates please implement your own playbooks/roles/tasks
+
+## Installation
+
+```bash
+ansible-galaxy collection install t_systems_mms.letsencrypt
+```
+
+## Providers
+
+The role supports multiple providers for http- and dns-challenges.
+Please see the corresponding readme files for specific variables and examples.
+
+Feel free to contribute with more DNS APIs or more HTTP variants :)
+
+### http-challenge
+
+Following providers for a http-challenge are currently supported.
+
+| Provider                         | Description
+|----------------------------------|-------------
+| [s3](http-challenge/s3.md)       | uses a aws s3 bucket for validation/hash files
+| [local](http-challenge/local.md) | uses a local path at your server/machine for validation/hash files
+
+### dns-challenge
+
+Following providers for a dns-challenge are currently supported.
+
+| Provider                                | Description
+|-----------------------------------------|-------------
+| [AutoDNS](dns-challenge/autodns.md)     |
```

Comment:
How about we add links to the corresponding docs here?

Reply:
I think this is a good idea, I see no benefit in the table here. Maybe convert this to a list of Links, that keeps it simple

Reply:
Yes, table can be removed. But i don't quite get what you mean with links? The entry is a link?

Reply:
Merge the providers into one list.

Reply:
I meant links to the upstream documentation of the providers. But this is fine, too!